apps/web/src/components/invoice-generator/index.tsx

```diff
@@ -188,7 +188,7 @@ function InputRowArray({
           {rows.map((row, index) => (
             <div key={row} id={`${htmlId}.${index.toString()}`}>
               <h3>
-                {t("Product")} {index + 1}
+                {t("Attachment")} {index + 1}
```

Comment:
Does this now change all to Attachment? Even the products? Might have to pass it as a prop more dynamically 🤔

Reply:
Ah, I must have misunderstood the original problem (that we discussed in Telegram), my bad. This should be fixed now.